Python/Watermark_Images/README.md

```diff
@@ -0,0 +1,16 @@
+# Watermark Images
+
+## Description:
+Simple python scripts that can be used to add watermark to an image using either text or logo.
+
+## Requirements to run the notebook:
+
+* OpenCV
+* Numpy
+* Python Imaging Library(PIL)
+
+## Examples:
+
+![image](https://i.ibb.co/DRXgKT2/Foodcover.png)
+
+![img](https://i.ibb.co/YyHyhDR/SkyCover.png)
```

Comment:
Add your name as Author as well, you have worked hard on this one!!!

PS: Follow the [README Template](https://github.com/HarshCasper/Rotten-Scripts/blob/master/TEMPLATE_README.md)

Reply:
Done!